content/en/docs/Concepts/operators-on-cluster.md

```diff
@@ -0,0 +1,21 @@
+# When to use operators on a cluster
+
+
+Operators are good for automating the knowledge of how to operate a complex system. Operators are a high-privilege component and by design, they run persistently inside your cluster. An operators can handle features like automatic scaling in response to load, backup and restore, integration with metric systems like Prometheus, and even failure detection and automatic tuning in response to usage patterns.
+
+# How does OLM help you use operators on a cluster?
+
+OLM requires you to provide metadata about your operator in order to ensure that it can be kept running safely on a cluster, and to provide information about how updates should be applied as you publish new versions of your operator. A ClusterServiceVersion (CSV) represents a particular version a running operator on a cluster. It includes metadata such as name, description, version, repository link, labels, icon, etc. It declares `owned`/`required` CRDs, cluster requirements, and install strategy that tells OLM how to create required resources and set up the operator as a [deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/).
+
+As a cluster administrator, you can install an Operator from the OperatorHub using the OpenShift Container Platform web console or the CLI. You can then subscribe the Operator to one or more namespaces to make it available for developers on your cluster.
```

Comment:
Is there another way to install the operator from OperatorHub? or does it require the OCP web console? These are upstream documents so the first option should be how someone would install their operator on a Kubernetes cluster using OLM. Then mention "or you can use the OpenShift Container Platform web console".